src/main/webapp/homepage_script.js

```diff
@@ -15,6 +15,22 @@
 const MAPSKEY = config.MAPS_KEY
 let neighborhood = [null , null];
 
+window.onscroll = stickyControlBar;
+
+/* Scroll function so that the control bar sticks to the top of the page */
+function stickyControlBar() {
+    const controlBarWrapper = document.getElementById("control-bar-message-wrapper");
+    const taskListDiv = document.getElementById("tasks-list");
+    const offset = 165;
```

Comment:
Common javascript syntax would be to put constants in all caps to differentiate them from regular variables.

Reply:
You're right, I'll update this, thanks!